### PR TITLE
You can say you to me

### PR DESCRIPTION
**Pull request by chrbeckm**
Changes to the wording in linux and macos installation notes, not in the Python part #230

### CODE REVIEW

File: install/linux.md [hunk 1]
```diff
@@ -60,20 +63,24 @@ Lizenz und enthalten Tracking Software. Deswegen nutzen wir VSCodium.
...
+  dem Software-Center (Doppelklick auf den Download). Klicke auf `Installieren`.
 
 
 - Arch Linux
 
       $ sudo pacman -S code
 
-Startet VS Code nach der Installation, klickt in der linken Leiste auf `Extensions` (Unterstes Symbol) und sucht nach „LaTeX Language Support“, installiert die Extension mit einem Klick auf den `install` Knopf.
+Starte VS Codium nach der Installation, klicke in der linken Leiste auf
+`Extensions` (Unterstes Symbol) und suche nach „TexLab LSP Client“,
```
Comment (aknierim) [inline install/linux.md:82]: lower case "unterstes" looks better to me

File: install/macos.md [hunk 2]
```diff
@@ -4,15 +4,18 @@ title: macOS-Installation
 ---
 
 
-Wir empfehlen die Installation einer 64-bit Version, falls man ein 64-bit Betriebssystem benutzt.
-Wenn man sich nicht sicher ist, hat man heutzutage fast immer 64-bit.
+Wir empfehlen die Installation einer 64-bit Version,
+wenn du ein 64-bit Betriebssystem benutzt.
+Wenn du dir nicht sicher bist, heutzutage ist fast jedes System 64-bit.
```
Comment (aknierim) [inline install/macos.md:9]: How about: "Wenn du dir dabei nicht sicher bist: Heutzutage ist fast jedes System 64-bit."

File: install/linux.md [hunk 3]
```diff
@@ -5,10 +5,12 @@ title: Linux-Installation
...
 __Es müssen alle Schritte (Installieren, Testen, Aktualisieren) ausgeführt werden!__
 
-Falls man nicht am LaTeX-Kurs teilnimmt, ist TeXLive optional.
-Dann kann man allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
+Falls du nicht am LaTeX-Kurs teilnimmst, ist TeXLive optional.
+Dann kannst du allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
 
-Falls man nur am LaTeX-Kurs teilnehmen will, sollte man mindestens Visual Studio Codium (oder einen anderen Texteditor) und TeXLive installieren.
+Falls du nur am LaTeX-Kurs teilnehmen willst,
+solltest du mindestens Visual Studio Codium (oder einen anderen Texteditor)
```
Comment (aknierim) [inline install/linux.md:12]: For consistency: Either use "VS Codium" here or use "Visual Studio Codium" in the macOS instructions

File: install/macos.md [hunk 4]
```diff
@@ -4,15 +4,18 @@ title: macOS-Installation
...
 __Es müssen alle Schritte (Installieren, Testen, Aktualisieren) ausgeführt werden!__
 
-Falls man nicht am LaTeX-Kurs teilnimmt, ist MacTeX optional.
-Dann kann man allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
+Falls du nicht am LaTeX-Kurs teilnimmst, ist MacTeX optional.
+Dann kannst du allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
 
-Falls man nur am LaTeX-Kurs teilnehmen will, sollte man mindestens VS Codium (oder einen anderen Texteditor) und MacTeX installieren.
+Falls du nur am LaTeX-Kurs teilnehmen willst,
+solltest du mindestens VS Codium (oder einen anderen Texteditor)
```
Comment (aknierim) [inline install/macos.md:17]: As mentioned before in my comment on the Linux instructions: "VS Codium" vs "Visual Studio Codium"

**Review by aknierim — CHANGES_REQUESTED** [review]

File: install/macos.md (hunk 4 above)
Comment (chrbeckm) [reply install/macos.md]: I changed it to VSCodium, as per their naming.

File: install/dualboot.md [hunk 5]
```diff
@@ -79,49 +80,55 @@ Alternativen zu Ubuntu sind zum Beispiel (alphabetisch):
...
+Wenn dies nicht geändert wird, stimmen die Uhrzeiten auf Windows und Linux nicht überein
+und können in beiden System falsch sein.
+
 
 Um Windows ebenfalls auf UTC umzustellen, muss die Registry bearbeitet werden.
 
-1. Öffnet das Programm `regedit`, am einfachsten, indem ihr `regedit` tippt,
-während das Startmenü offen ist.
+1. Öffne das Programm `regedit`; am einfachsten, indem du `regedit` tippst,
+  während das Startmenü offen ist.
```
Comment (aknierim) [inline install/dualboot.md:104]: "während das Startmenü offen ist, und klicke auf ``Registrierungs-Editor``."

File: install/dualboot.md [hunk 6]
```diff
@@ -131,35 +138,42 @@ Also tatsächlich herunterfahren, den Stick einstecken,
...
 
-1. Ihr solltet als Sprache Englisch wählen, aus dem einfachen Grund, dass der Großteil
-der Online-Dokumentation auf Englisch sind.
+1. Aktiviere die Optionen `Install Third Party Software` und falls du mit einem
+  Netzwerk verbunden bist, installiert Updates während der Installation.
 
-1. Keyboard-Layout sollte zu eurem Laptop passen, also vermutlich `German (eliminate dead keys)`.
-Die `eliminate dead keys` Variante sorgt dafür, dass Akzente direkt erscheinen, was sehr hilfreich für
-LaTeX ist.
+1. Die Ubuntu Installation scanned nun deine Festplatte und sollte Windows und den
```
Comment (aknierim) [inline install/dualboot.md:165]: "scannt"

**Review by aknierim — CHANGES_REQUESTED** [review]

**Review by aknierim — CHANGES_REQUESTED** [review]
Except for me suggesting a somewhat weird sounding wording in the previous review, this looks good to me 👍

**Review by aknierim — APPROVED** [review]

**Pull request merged**